### PR TITLE
fix bug, add rpc nodes

### PR DESCRIPTION
**Pull request by gregcusack**
#### Summary of Changes
1) Fix bug in startup scripts
2) default to enable-warmup-epochs
3) fix secrets creation to set as file in pod
3) Create rpc accounts
4) create/deploy rpc secrets, selectors, services, replica sets

### CODE REVIEW

**Review by yihau — APPROVED** [review]
🪖

File: src/main.rs [hunk 1]
```diff
@@ -612,6 +638,86 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
...
+                    break;
+                }
+            }
+            if ready {
+                break;
+            }
+
+            info!("no rpc nodes ready yet");
+            tokio::time::sleep(std::time::Duration::from_secs(1)).await;
+        }
```
Comment (joncinque) [inline src/main.rs:718]: micro-nit: you could use a labled loop for this to be a bit cleaner, ie:

```
        'outer: loop {
            for rpc_name in &rpc_nodes {
                if kub_controller.is_replica_set_ready(rpc_name).await? {
                    break 'outer;
                }
            }

            info!("no rpc nodes ready yet");
            tokio::time::sleep(std::time::Duration::from_secs(1)).await;
        }
```

More info at https://doc.rust-lang.org/rust-by-example/flow_control/loop/nested.html

**Review by joncinque — COMMENTED** [review]
Looking good! Just a few little things

File: README.md [hunk 2]
```diff
@@ -81,6 +80,12 @@ cargo run --bin cluster --
 --metrics-password <metrics-password>   # from (1)
 ```
 
+#### RPC Nodes
+You can add in RPC nodes. These sit behind a load balancer. Load balancer distributed loads across all RPC nodes that the bootstrap. Set the number of RPC nodes with:
```
[the remark this answers is not in the document]
Comment (gregcusack) [reply README.md:84]: lol typo. Should say "Load balancer distributed loads across all RPC nodes AND the bootstrap"

[the remark this answers is not in the document]
Comment (gregcusack) [reply README.md:84]: side note, one of the things I need to do is draw a diagram of how this all works and push it to this readme

File: src/startup_scripts.rs [hunk 3]
```diff
@@ -30,10 +29,25 @@ nohup solana-faucet --keypair bootstrap-accounts/faucet.json &
 # shellcheck disable=SC1091
 source /home/solana/k8s-cluster-scripts/common.sh
 
-program="agave-validator"
-
 no_restart=0
 
+# Define the paths to the validator cli. pre 1.18 is `solana-validator`. post 1.18 is `agave-validator`
+agave_validator="/home/solana/.cargo/bin/agave-validator"
+solana_validator="/home/solana/.cargo/bin/solana-validator"
```
[the remark this answers is not in the document]
Comment (gregcusack) [reply src/startup_scripts.rs:36]: ya you're right lol. it was just the way it was done in the `net/` scripts so kept it the same for familiarity. but i can change it no prob

File: src/main.rs (hunk 1 above)
Comment (gregcusack) [reply src/main.rs:718]: ohhh lets goooo. this is much better. thank you!

File: src/startup_scripts.rs (hunk 3 above)
[the remark this answers is not in the document]
Comment (joncinque) [reply src/startup_scripts.rs:36]: Oh gotcha, I didn't realize they were put there in the `net/` -- in that case, feel free to save it for later!

[the remark this answers is not in the document]
Comment (gregcusack) [reply src/startup_scripts.rs:36]: honestly I didn't like that they were in /home/solana/.cargo/ anyway, so going to update as you've suggested

**Pull request merged**